src/NServiceBus.Core/Pipeline/Incoming/ReceivePipelineCompleted.cs

```diff
@@ -0,0 +1,40 @@
+namespace NServiceBus
+{
+    using System;
+    using Transport;
+
+    /// <summary>
+    /// The ReceivePipeline completed event.
```

Comment:
How about something

> Indicates when a receive pipeline is completed. 

Should we also indicate in the doco here that this event is only raised for the shared queue and not or the instance specific one?